### ISSUE TITLE
Create feature tests for testing UI and js

### ISSUE THREAD

**Issue by davydovanton** [ossboard-org/ossboard#53]
Will be awesome to test our UI and js code. We can use capybara for that.

What we need to test primarily:
* Web app
  * New task page (create task, get issue)
* Admin app
  * Moderation page (approve task or not)
  * User page (block, unblock users)

**Comment by davydovanton** [comment]
What we need to test:
1. Switcher on index page (http://www.ossboard.org)
2. Task status filter on task page (http://www.ossboard.org/tasks)
3. Export logic on new page (http://www.ossboard.org/tasks/new)
4. Preview switcher on new page (http://www.ossboard.org/tasks/new)

**Comment by moofkit** [comment]
@davydovanton I can take this issue.
But I have some questions:
1. What driver we can use? Each driver has pros and cons. I.e. phantomjs is fast, but some times test is failed without any resons. Firefox/chrome is slow, but it's close to real using.
2. Can I use some abstraction layer for capybara? I'm talking about https://github.com/natritmeyer/site_prism. The main idea that you have an page object with declaritive description of testing page or pages and easy-breeze test cases. When structure of page is changed and test's are failed, you need to change the page objects not tests.

**Comment by moofkit** [comment]
Also have an issue with configuration of capybara.
And this is reason https://github.com/ossboard-org/ossboard/blob/master/spec/support/capybara.rb#L5

I've changed it to ::Hanami::App.new as explainde here https://github.com/davydovanton/rspec-hanami/issues/1
but then i have another issue `app.rb:21:in 'initialize': wrong number of arguments (given 0, expected 2) (ArgumentError)`
That seems reasonable https://github.com/hanami/hanami/blob/master/lib/hanami/app.rb#L22
But how configure this properly?

**Comment by inpego** [comment]
I'm already working on this task using phantomjs and poltergeist. Thank you for attracting my attention to site_prism.

**Comment by moofkit** [comment]
@inpego ok! If you have some questions about site_prism or how to use it you can ask me.

btw how you solved this issue with `capybara.app`? :)

**Comment by inpego** [comment]
I had no problems with the capybara, try checkout my branch and run tests.
